docs/application/dotnet/samples/location/overview.md

```diff
@@ -57,6 +57,15 @@ for TD:
 <div class="tabcontent" id="Wearable">
 <table>
 	<tbody>
+		<tr>
+			<td>
+			<p><img alt="" height="180" src="media/w68findplace.png" width="180" /></p>
+			</td>
+			<td>
+                        <p><strong>(W) FindPlace</strong> [In progress]</p>
+			<p>This sample application demonstrates how to search for places within a specified area using <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.Maps.html" target="_blank">Tizen.Maps</a>.</p> and how to create a circular UI using <a href="https://samsung.github.io/Tizen.CircularUI/api/index.html" target="_blank">Tizen.Wearable.CircularUI</a>.</p>
```

Comment:
Please make this small change, as I see a small gap between the lines and there is a period in between the sentence. I have removed the period and /P
Thanks in advance.

This sample application demonstrates how to search for places within a specified area using <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.Maps.html" target="_blank">Tizen.Maps</a> and how to create a circular UI using <a href="https://samsung.github.io/Tizen.CircularUI/api/index.html" target="_blank">Tizen.Wearable.CircularUI</a>.

Reply:
Fixed it.